Mathlib/Analysis/Fourier/PoissonSummation.lean

```diff
@@ -174,9 +174,9 @@ theorem isBigO_norm_Icc_restrict_atBot {f : C(ℝ, E)} {b : ℝ} (hb : 0 < b)
   rintro ⟨x, hx⟩
   rw [ContinuousMap.restrict_apply_mk]
   refine' (le_of_eq _).trans (ContinuousMap.norm_coe_le_norm _ ⟨-x, _⟩)
-  rw [ContinuousMap.restrict_apply_mk, ContinuousMap.comp_apply, ContinuousMap.coe_mk,
-    ContinuousMap.coe_mk, neg_neg]
-  exact ⟨by linarith [hx.2], by linarith [hx.1]⟩
+  · rw [ContinuousMap.restrict_apply_mk, ContinuousMap.comp_apply, ContinuousMap.coe_mk,
+      ContinuousMap.coe_mk, neg_neg]
+    exact ⟨by linarith [hx.2], by linarith [hx.1]⟩
```

Comment:
Does this really deserve to be indented?

Reply:
At that point, there are two goals, the second being
```lean
-x ∈ Icc (-x✝ + -S) (-x✝ + -R)
```
I do not understand though how Lean solves this one.

Reply:
Oh, I think that I see now

The "first goal" becomes identical to the second one after the `rw`.  So, I guess that the final `exact` solves the current goal and then Lean realizes that it already knows how to solve that goal and closes the second one as well.